contentgrid-junit-jupiter-k8s/src/main/java/com/contentgrid/junit/jupiter/k8s/KubernetesTestClusterExtension.java

```diff
@@ -68,6 +71,19 @@ Store getStore(ExtensionContext context) {
         return context.getStore(NAMESPACE);
     }
 
+    // Ensure the DockerRegistryCache extensions runs first, if the test class is annotated by both
+    void ensureRegistryCacheFirst(ExtensionContext context) throws Exception {
+        var annotations = AnnotationSupport.findRepeatableAnnotations(context.getRequiredTestClass(), DockerRegistryCache.class);
+        if (annotations.isEmpty()) {
+            return;
+        }
+        var mirrors = DockerRegistryCacheExtension.getMirrors(context);
+        if (mirrors == null || mirrors.size() == 0) {
```

Comment:
mirrors cannot be null (getMirrors is annotated)
you can replace this by mirrors.isEmtpy()

I got these remarks from my IDE